src/sentry_plugins/slack/plugin.py

```diff
@@ -152,18 +166,18 @@ def notify(self, notification):
 
         # They can be the same if there is no culprit
         # So we set culprit to an empty string instead of duplicating the text
-        if culprit and title != culprit:
+        if self.get_option('include_culprit', project) and culprit and title != culprit:
```

Comment:
both of these will change the behavior for people who have already configured their slack integrations since `self.get_option('include_culprit', project)` will return `None`. can you please adjust your changes so that this won't affect existing slack users?